UKHomeOffice/nmsw-ui
# NMSW-38 Add Dockerfile and NGINX setup

# Ticket

NMSW-38

----

## AC

Add dockerfile, run.sh, and nginx.config based on how it was implemented for sPCR

----

## To test

* build `docker build -t nmsw-ui .`
* run `docker run -d --name nmswui -p 8080:8080 nmsw-ui .`
* go to localhost:8080
* > you should see a page with a title, some text, and a button in the GDS styling

----

## Notes
